Cython/Utility/ImportExport.c

```diff
@@ -329,17 +330,20 @@ static int ${import_star}(PyObject* m) {
     PyObject *utf8_name = 0;
     PyObject *name;
     PyObject *item;
+    PyObject *import_obj;
 
     locals = PyDict_New();              if (!locals) goto bad;
     if (__Pyx_import_all_from(locals, m) < 0) goto bad;
     list = PyDict_Items(locals);        if (!list) goto bad;
 
-    for(i=0; i<PyList_GET_SIZE(list); i++) {
-        name = PyTuple_GET_ITEM(PyList_GET_ITEM(list, i), 0);
-        item = PyTuple_GET_ITEM(PyList_GET_ITEM(list, i), 1);
+    for(i=0; i<PyList_Size(list); i++) {
+        import_obj = __Pyx_PyList_GET_ITEM(list, i); if (!import_obj) goto bad;
+        name = __Pyx_PyTuple_GET_ITEM(import_obj, 0); if (!name) goto bad;
+        item = __Pyx_PyTuple_GET_ITEM(import_obj, 1); if (!item) goto bad;
+
         utf8_name = PyUnicode_AsUTF8String(name);
         if (!utf8_name) goto bad;
-        s = PyBytes_AS_STRING(utf8_name);
+        s = __Pyx_PyBytes_AsWritableString(utf8_name); if (!s) goto bad;
```

Comment:
```suggestion
        s = __Pyx_PyBytes_AsString(utf8_name); if (!s) goto bad;
```

I think you've used `AsWriteableString` to match the `char*` type.

Assuming https://github.com/cython/cython/pull/6101 doesn't break anything unexpected, I'll change it to `const char*` and we should just use `AsAtring` (since it isn't something that gets written to)

Reply:
Yup that's right, I originally had AsString but changed it to match the current usage of it. Happy to change it if that works.